README.md

```diff
@@ -217,6 +227,217 @@ bash terraform/modules/migrations/run-migrations.sh
 
 Already ran migrations will not be ran again.
 
+### 7. Enable unauthenticated PostgREST invocations (optional)
+
+If you are using a branch that by default requires authentication for PostgREST,
+like the `emojicoin-dot-fun` branch, you'll need to select
+`Allow unauthenticated invocations` under `Cloud Run > postgrest > security`.
+
+### 8. Disable WebSockets certificate checking (optional)
+
+If you run `inbox` via Docker compose, the events will emit over both the `mqtt`
+protocol (`mqtt://localhost:21833`) and the `ws` protocol
+(`ws://localhost:21884`), but you can't connect to an `mqtt` channel in the
+browser.
+
+Since some browsers, like Chrome, require certificate authority (CA) checks, it
+is suggested that you simply disable CA checks at your browser if necessary so
+that you can use the `ws` channel (instead of `wss`). For example to disable
```

Comment:
Actually not sure if this is ws or wss, it might be using wss it just doesn't check if it's actually secure since it's local. Will have to check tomorrow

Reply:
Addressed in a1ef2c0 per eng call

Reply:
Resolving per eng call